### PR TITLE
Fixing broken link to Paul Bloch of openartist

### PR DESCRIPTION
**Pull request by deborahtrez**
This pull request fixes issue #200 
Broken link to Paul Bloch of openartist.

Kindly review :) 
@ywwg

### CODE REVIEW

File: content/news/2008-02-15-minor-website-revamp.md [hunk 1]
```diff
@@ -10,12 +10,12 @@ If anyone reads our developer's mailing list, you might know I'm not a huge fan
...
 Our website is now located at: **[www.mixxx.org]({filename}/pages/index.md)**
 
-We've also started using our new logo, created by [Paul Bloch of OpenArtist](http://openartist.net/).
+We've also started using our new logo, created by [Paul Bloch of OpenArtist](https://git.opendesktop.org/openartist/).
 He's quite talented, and is working on a more thorough overhaul of the Mixxx website to be launched sometime in the upcoming months.
 
 Among the new changes to the website are our [new forums](http://www.mixxx.org/forums), and our new (and faster) [Mixxx
 Wiki](http://github.com/mixxxdj/mixxx/wiki).
-We're looking forward to helping the community grow on the new forums.
+We are looking forward to helping the community grow on the new forums.
```
Comment (Holzhaus) [inline content/news/2008-02-15-minor-website-revamp.md]: I think this change is unnecessary.

**Review by Holzhaus — COMMENTED** [review]
Thanks.

**Comment by Holzhaus** [comment]
Please remove the second commit with `git reset HEAD~1`.

You can add `Fixes #200.` to the commit message of the first commit using `git commit --amend` and make the commit message look like this:

```
Corrected link to Paul Bloch of openArtist

Fixes #200.
```

**Comment by Holzhaus** [comment]
@deborahtrez I don't know what you did (rebased?), but now the commits in this branch look broken.

I suggest to reset this branch to the upstream website branch, then pick the actual commit from the reflog.
```
git fetch upstream
git reset --hard upstream/website
git cherry-pick c03d7f947f8f80af722604f96a450185407efc9a
```

**Comment by deborahtrez** [comment]
> @deborahtrez I don't know what you did (rebased?), but now the commits in this branch look broken.
> 
> I suggest to reset this branch to the upstream website branch, then pick the actual commit from the reflog.
> 
> ```
> git fetch upstream
> git reset --hard upstream/website
> git cherry-pick c03d7f947f8f80af722604f96a450185407efc9a
> ```
Thank you.

I was trying to squash the commits into one but used git fixup instead. I was using Git interactive rebase for the first time.

**Review by Holzhaus — APPROVED** [review]
LGTM, thank you.

**Pull request merged**